ios/en.lproj/InfoPlist.strings

```diff
@@ -1,5 +1 @@
 "NSHumanReadableCopyright" = "Copyright © 2020 Path Check Inc. All rights reserved.";
```

Comment:
There are also a couple of notification strings for when the app is closed on the `Localizable.strings` file we might want to remove, more details on [trello](https://trello.com/c/Imm1CVLo)